app/controllers/chores_controller.rb

```diff
@@ -0,0 +1,37 @@
+class ChoresController < ApplicationController
+  def create
+    begin
+      chore = Chore.create(chore_params)
```

Comment:
Let's not pass in the entire object like this, just stick with individualized parameters `ex. Chore.create(name: name, etc)`, it improves readability quite a bit.